parlai/mturk/core/scripts/approve_hits.py

```diff
@@ -0,0 +1,157 @@
+import argparse
```

Comment:
@bhancock8 should this file be added? @JackUrb ?

Reply:
No, this was purely a hacked together script for approving a bunch of HITs that I lost the handles to because of crashes. No reason to add it.